sequelize/migrations/correlation/04_correlations.js

```diff
@@ -0,0 +1,121 @@
+'use strict';
+
+// See manual:
+// https://sequelize.readthedocs.io/en/latest/docs/migrations/#functions
+
+// CHANGE NOTES:
+// Changes between 1.2.0 and 2.0.0:
```

Comment:
This pattern is actually somewhat obsolete. 
I think from now on, we should just remark the version of the repository that contains the changes.

Like so:

```suggestion
// Changes for version <Insert new version of correlation repository here> 
```